scripts/lintChanged.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Checks for changes between the common ancestor of origin/main and the
+# current HEAD. 
+
+source scripts/shellUtils.sh
+
+MERGE_BASE_SHA_HASH=$(git merge-base origin/main HEAD)
```

Comment:
I think you will need to use the `curl` command you suggested on the issue to get the merge base if you are not doing a full clone. Also be sure to quote variable assignments for shell safety. (Recommend running `shellcheck` against your script in general, it will catch a lot of these comments)

```suggestion
MERGE_BASE_SHA_HASH="$(git merge-base origin/main HEAD)"
```

Reply:
I don't think I have any special configs when I run it locally on mine 🤔

Reply:
Resolved in c8987b37cea6a635afecac06ced957e2540ae043